### PR TITLE
Adding 028 clojure solution

### PR DESCRIPTION
**Pull request by fredericojordan**
### How the solution works

[Problem 028 on wiki](/FrankKair/polyglot-euler/wiki/Problem-028)

### Performance

[Try it online!](https://tio.run/##ZY7BCsMgEETvfsUc3ZaChtJDoV8SQgjRlhSrQUnw761KSSDdy@zsPHZ3NO69eJ0SV/ppEebJD6ZX0/Dqw/JBGzsG8NFZlTV3D0RIgtxdk52o9q5N0OBn8BOuOYlUx1vxS0luRcsSOsbH45VrqBRjv/@cWTXa7h@WQsjKzd7mtHBEKX0B)

```
Real time: 3.211 s
User time: 4.961 s
Sys. time: 0.443 s
CPU share: 168.31 %
Exit code: 0
```

### CODE REVIEW

File: src/028/p028.clj [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  (cond
+    (= x 1) 1
+    (= x 2) 10
+    :else (+ (* 4 x x)
+             (- (* 6 (- x 1)))
+             (spiral_diag_sum (- x 2)))))
+
+(defn solve [] (spiral_diag_sum 1001))
+
+(prn (solve))
```
Comment (FrankKair) [inline src/028/p028.clj]: What's the difference between `prn` and `println`?

File: src/028/p028.clj [hunk 2]
```diff
@@ -0,0 +1,11 @@
+(defn spiral_diag_sum [x]
```
Comment (FrankKair) [inline src/028/p028.clj]: I'm not sure, but I think that Clojure uses "kebab-case", like so `spiral-diag-sum`.
Can you check?

Comment (fredericojordan) [reply src/028/p028.clj]: TBH, didn't even look, just copied the function name from my elixir solution! 😂
But you're right, I'll update it! Thanks

Comment (FrankKair) [reply src/028/p028.clj]: Haha, ok 😊

**Pull request merged**